docs/modules/ROOT/pages/documentation/ecosystem/maven-plugin.adoc

```diff
@@ -317,6 +320,43 @@ the process id of the running Payara Micro instance.
 
 |===
 
+=== `reload`
+*Aim*: This goal reloads the Payara Micro application with the specified configuration.
+By default, this goal performs the vanilla deployment of the application without restarting the 
+Payara Micro instance. If `hotDeploy` is set to true then the Hot Reload functionality
+ will be enabled. 
+
+NOTE:
+This goal is specifically designed for integration with the IDEs and Editors.
```

Comment:
```suggestion
NOTE: This goal is specifically designed for integration with the IDEs and Editors.
```
The NOTE word must be on the same line, otherwise it's not rendered as expected.